src/bandersnatch/tests/plugins/test_allowlist_name.py

```diff
@@ -406,6 +406,43 @@ def test__filter__matches__release(self) -> None:
 
         self.assertEqual({"1.2.0": {}}, pkg.releases)
 
+    def test__filter__matches__release_latest(self) -> None:
+        with open(Path(self.tempdir.name) / "requirements.txt", "w") as fh:
+            fh.write("""\
+#    This is needed for workshop 1
+#
+foo==1.2.0             # via -r requirements.in
+""")
+
+        mock_config(f"""\
+[mirror]
+storage-backend = filesystem
+workers = 2
```

Comment:
Is there a reason two have 2 workers here? (It's fine, just interested)

Reply:
again just copy paste from the unit test above (removed with last commit)